doc/manual.css

```diff
@@ -0,0 +1,102 @@
+/*
+Stylesheet for Docutils, based on voidspace.css.
+Based on ``blue_box.css`` by Ian Bicking
+and ``html4css1.css`` revision 1.46.
+*/
+
+@import url(html4css1.css);
```

Comment:
Where is this file and do we need it?

Reply:
It comes with rst2html. It probably depends where we want to build it as to whether we need to duplicate the file or what.

Reply:
Okay.